src/pages/dashboard/[[...id]].js

```diff
@@ -105,18 +118,20 @@ function Country({ country: countrySlug, data, errorCode, ...props }) {
         alignItems="center"
         container
       >
-        <Grid
-          item
-          lg={12}
-          id="map"
-          className={`${classes.section} ${classes.topMargin}`}
-        >
-          <SensorMap
-            zoom={COUNTRIES_LOCATION[country].zoom}
-            latitude={COUNTRIES_LOCATION[country].latitude}
-            longitude={COUNTRIES_LOCATION[country].longitude}
-            location={COUNTRIES_LOCATION[country].label}
-          />
+        <Grid container direction="row" className={classes.mapSection}>
```

Comment:
Why do we need this `Grid`? We should be able able to add {classes.mapSection} to the Grid item below i.e.
```jsx
          <Grid
            item
            xs={12}
            id="map"
            className={`${classes.section} ${classes.topMargin} ${classes.mapMargin}`}
          >
            ...
```

PS: Is it confirmed that the product owner doesn't want the map to go end to end?